hummingbot/connector/exchange/btc_markets/btc_markets_websocket.py

```diff
@@ -0,0 +1,121 @@
+#!/usr/bin/env python
+import asyncio
+import logging
+import websockets
+import ujson
+import hummingbot.connector.exchange.btc_markets.btc_markets_constants as constants
+from hummingbot.core.utils.async_utils import safe_ensure_future
+
+
+from typing import Optional, AsyncIterable, Any, List
+from websockets.exceptions import ConnectionClosed
+from hummingbot.logger import HummingbotLogger
+from hummingbot.connector.exchange.btc_markets.btc_markets_auth import BtcMarketsAuth
+from hummingbot.connector.exchange.btc_markets.btc_markets_utils import RequestId, get_ms_timestamp
+
+# reusable websocket class
+# ToDo: We should eventually remove this class, and instantiate web socket connection normally (see Binance for example)
```

Comment:
Yes, please, remove this class.

Reply:
picked up from crypto.com connector - https://github.com/hummingbot/hummingbot/commit/6139d474d988200f868bb886375110d55afc0c12